gitlab/http_backends/requests_backend.py

```diff
@@ -39,6 +39,38 @@ def __init__(self, session: Optional[requests.Session] = None) -> None:
     def client(self) -> requests.Session:
         return self._client
 
+    @staticmethod
+    def prepare_send_data(
```

Comment:
I think we should make these things private somehow before we release this, maybe not here but by making the client backends private or so. Not urgent for now.